apps/OboeTester/app/src/main/java/com/mobileer/oboetester/StreamConfiguration.java

```diff
@@ -130,6 +147,12 @@ public StreamConfiguration() {
         for (int usage : usages) {
             mUsageStringToIntegerMap.put(convertUsageToText(usage), usage);
         }
+
+        // Build map for Content Type string-to-int conversion.
+        mContentTypeStringToIntegerMap = new HashMap<String,Integer>();
```

Comment:
Add:
    mContentTypeStringToIntegerMap.put(convertContentTypeToText(UNSPECIFIED), UNSPECIFIED);

Do same for Usage.

Reply:
Great idea!